src/pages/TimetableManagement/page.tsx

```
@@ -38,6 +45,38 @@ export default function Page() {
 
+    const column: ColumnInferface = {
+        count: 3,
+        width: [
+            '20px',
+            'calc((100% - 20px) / 2)',
+            'calc((100% - 20px) / 2)'
+        ],
+        headNode: [
+            ...COLUMN_NAME_LIST.map((columnName) => (
+                <TableHeader>
+                    <TableHeaderText color={colors.gray70}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)